maplibre_gl_example/lib/animate_camera.dart

```diff
@@ -122,6 +144,22 @@ class AnimateCameraState extends State<AnimateCamera> {
                   },
                   child: const Text('scrollBy'),
                 ),
+                TextButton(
+                  onPressed: () {
+                    mapController.queryCameraPosition().then(
+                          (result) => debugPrint(
+                              "queryCameraPosition() returned $result"),
+                        );
+                  },
+                  child: const Text('queryCameraPosition'),
+                ),
```

Comment:
`queryCameraPosition` isn't available on web, disable this sample when `kIsWeb`
```suggestion
                if(!kIsWeb)
                    TextButton(
                    onPressed: () {
                        mapController.queryCameraPosition().then(
                            (result) => debugPrint(
                                "queryCameraPosition() returned $result"),
                            );
                    },
                    child: const Text('queryCameraPosition'),
                    ),
```